generators/server/templates/src/main/java/package/config/_WebConfigurer.java

```diff
@@ -53,13 +54,13 @@ public void onStartup(ServletContext servletContext) throws ServletException {
         EnumSet<DispatcherType> disps = EnumSet.of(DispatcherType.REQUEST, DispatcherType.FORWARD, DispatcherType.ASYNC);<% if (clusteredHttpSession == 'hazelcast') { %>
         initClusteredHttpSessionFilter(servletContext, disps);<% } %>
         initMetrics(servletContext, disps);
-        if (env.acceptsProfiles(Constants.SPRING_PROFILE_PRODUCTION)) {
-            initCachingHttpHeadersFilter(servletContext, disps);
-            initStaticResourcesProductionFilter(servletContext, disps);
-        }<% if (devDatabaseType == 'h2Disk' || devDatabaseType == 'h2Memory') { %>
-        if (env.acceptsProfiles(Constants.SPRING_PROFILE_DEVELOPMENT)) {
-            initH2Console(servletContext);
-        }<% } %>
+        if (env.acceptsProfiles(Constants.SPRING_PROFILE_PRODUCTION)) {<% if (!skipClient) { %>
```

Comment:
No worries, I didn't! I'm quite happy with the feedback, actually -- obviously I really desperately need the insight of jhipster veterans for something like this!